### DISCUSSION TITLE
Task retry and skipping from UI

### DISCUSSION THREAD

**Discussion by tri97nguyen** [PrefectHQ/prefect#15829] [Q&A]
I'm evaluating Prefect for my team but not sure how to retry a failed task from the UI? Or skip a failed task and continue the rest of the flow?
That seems like a very common operation but somehow I don't see it anywhere
Thank you

**Comment by desertaxle** [comment]
Hey @tri97nguyen! Thanks for opening a discussion!

Prefect can't currently retry a failed task run from the UI, but you can retry a flow run from the UI if the flow is associated with a [deployment](https://docs.prefect.io/3.0/deploy/index).

You could add the ability to retry or skip tasks using Prefect's [interactive workflow features](https://docs.prefect.io/3.0/develop/inputs). That would allow you to pause your workflow if a task fails and choose from the UI whether the task should be retried or skipped.

Let me know if you have any other questions about either of those options!
